### DISCUSSION TITLE
InodeCache.cpp:101:10: error: case value evaluates to 2435016766, which cannot be narrowed to type '__fsword_t' (aka 'int') [-Wc++11-narrowing]

### DISCUSSION THREAD

**Discussion by UllrichKossow** [ccache/ccache#1142] [Development]
With commit 5edcc6c36d881e0778c8d6b0b430a1ca054426d8 - "feat: Improve inode cache robustness" building ccache on a Raspberry Pi 4 (debian 10.12), using clang or gcc fails. This is the error message from clang:
/mnt/x/Projects/ccache/src/InodeCache.cpp:101:10: error: case value evaluates to 2435016766, which cannot be narrowed to type '__fsword_t' (aka 'int') [-Wc++11-narrowing]
    case 0x9123683e: // BTRFS_SUPER_MAGIC

Is this a configuration Problem?
I run cmake like this:
cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DREDIS_STORAGE_BACKEND=OFF
Looking on the statfs type definition, i found, this should be an issue on all 32 bit linux systems...
Thanks a lot.

**Comment by jrosdahl** [comment]
Thanks for the notice. Fixed in bed8a2c1e436a2b23fd2facd06b67cd98f629a32.

**Comment by UllrichKossow** [comment]
@jrosdahl  Thanks a lot!
